modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheGroupMetricsImpl.java

```diff
@@ -126,13 +133,22 @@ public CacheGroupMetricsImpl(CacheGroupContext ctx) {
             this::getTotalAllocatedSize,
             "Total size of memory allocated for group, in bytes.");
 
-        mreg.register("StorageSize",
-            this::getStorageSize,
-            "Storage space allocated for group, in bytes.");
+        if (ctx.shared().database() instanceof GridCacheDatabaseSharedManager) {
```

Comment:
Let's create `isPDSEnabled` method instead of direct usage `instanceof` operator

Reply:
Done